### ISSUE TITLE
Dialog buttons are not focusable

### ISSUE THREAD

**Issue by artem-zinnatullin** [afollestad/material-dialogs#211]
Dialog buttons (positive, negative, neutral) should be focusable because they don't respond to keyboard navigation (you can try it in emulator or by attaching bluetooth or usb keyboard to your device)

**Comment by afollestad** [comment]
This is now in the latest code, it will be included in the next release

**Comment by afollestad** [comment]
0.6.1.4 is now available

**Comment by afollestad** [comment]
Had to skip up to 0.6.1.5 due to issues with 0.6.1.4

**Comment by artem-zinnatullin** [comment]
@afollestad awesome, thanks!

**Comment by Alexandre-Nut** [comment]
Hi,

On 0.9.3.0 i can't focus Dialog buttons (positive, negative, neutral).
So my app need remote controler or gamepad...
Have you solution?

Thanks for your help.
